nautobot/ipam/views.py

```diff
@@ -80,7 +80,7 @@ class NamespaceUIViewSet(
     form_class = forms.NamespaceForm
     bulk_update_form_class = forms.NamespaceBulkEditForm
     filterset_class = filters.NamespaceFilterSet
-    queryset = Namespace.objects.all()
+    queryset = Namespace.objects.select_related("location")
```

Comment:
Should we prefetch tags here? It's a default column

Reply:
We actually have code for tables to automatically `prefetch_related` based on the visible table columns - https://github.com/nautobot/nautobot/blob/develop/nautobot/core/tables.py#L102-L131. It may be that the more optimal path here long-term is to address the TODO comment there and add logic to automatically handle `select_related` as well?

Reply:
Yeah that looks like a good idea

Reply:
I'll look into it.